libcxx/include/format

```diff
@@ -237,21 +237,21 @@ namespace std {
 #  include <cstdint>
 #  include <cstdlib>
 #  include <cstring>
-#  include <cwchar>
 #  include <initializer_list>
 #  include <limits>
+#  include <locale>
```

Comment:
The commit message doesn't say anything about why you've _also_ made the includes of `<locale>`, `<queue>` and `<stack>` unconditional. Was that change included in this patch by mistake? If it was on purpose, could you update the commit message to explain why it's related or necessary?

Reply:
We're not making them unconditional. If you expand this window to see the lines above, you'll see that the exact same conditional guards the includes above.

I merged the two sections to make the code clearer.

Reply:
As I think we'll loose or bury this convo if I force-push, I've updated the description in my automated first comment in this thread. I'll update it when I merge it. Unless there's a better way to handle this.